src/pages/dmt5-trading-signals/_signal-steps.tsx

```diff
@@ -26,24 +26,25 @@ const Container = styled.section`
 `
 const content = {
     subscriber: {
-        header: <Localize translate_text="How to subscribe to an MT5 signal" />,
-        step_one_header: <Localize translate_text="1. Click on the Signals tab" />,
+        header: <Localize translate_text="_t_How to subscribe to an MT5 signal_t_" />,
+        step_one_header: <Localize translate_text="_t_1. Click on the Signals tab_t_" />,
         step_one_text: (
-            <Localize translate_text="From your Deriv MT5 trading terminal, click on the Signals tab to view the list of signal providers." />
+            <Localize translate_text="_t_From your Deriv MT5 trading terminal, click on the Signals tab to view the list of signal providers._t_" />
         ),
         step_one_text_eu: (
-            <Localize translate_text="Go to your MT5 desktop app terminal and click on the Signals tab to view the list of signal providers. Not available on the web version." />
+            <Localize translate_text="_t_Go to your MT5 desktop app terminal and click on the Signals tab to view the list of signal providers. Not available on the web version._t_" />
         ),
-        step_two_header: <Localize translate_text="2. Subscribe to a signal provider" />,
+        step_two_header: <Localize translate_text="_t_2. Subscribe to a signal provider_t_" />,
         step_two_text: (
-            <Localize translate_text="Select the signal provider you prefer and click the Subscribe button." />
+            <Localize translate_text="_t_Select the signal provider you prefer and click the Subscribe button._t_" />
         ),
-        step_three_header: <Localize translate_text="3. Configure the parameters" />,
+        step_three_header: <Localize translate_text="_t_3. Configure the parameters_t_" />,
         step_three_text: (
-            <Localize translate_text="Configure your trading and risk management parameters. Then click OK to complete the process." />
+            <Localize translate_text="_t_Configure your trading and risk management parameters. Then click OK to complete the process._t_" />
```

Comment:
We have to refactor it, we can't use `Localize` components inside objects/arrays
![Screenshot 2023-03-06 at 10 25 39](https://user-images.githubusercontent.com/103182473/223051997-8d4e5979-2e4e-4df6-8416-adcffe7d2708.png)